support-frontend/assets/helpers/forms/paymentIntegrations/readerRevenueApis.ts

```diff
@@ -63,7 +63,13 @@ type SupporterPlus = {
 	amount: number;
 	currency: string;
 	billingPeriod: BillingPeriod;
-	fulfilmentOptions?: FulfilmentOptions;
+};
+export type TierThree = {
+	productType: 'TierThree';
+	amount: number;
```

Comment:
I don't think we need `amount` do we? This is fixed for T3

Reply:
Correct and rectified.